src/deploy_tools/demo_configuration/argocd/0.1.yaml

```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=/workspaces/deploy-tools/src/deploy_tools/models/schemas/release.json
+
+module:
+  name: argocd
+  version: "0.1"
+  description: Demonstration of binary download
+
+  applications:
+    - app_type: binary
+      name: argocd
+      url: https://github.com/argoproj/argo-cd/releases/download/v2.14.10/argocd-linux-amd64
+      hash_type: sha256
```

Comment:
What do you think of defaulting to sha256?

Reply:
I did that originally, then decided it was unclear in the yaml what was happening.